### ISSUE TITLE
Show renamed options properly

### ISSUE THREAD

**Issue by ncfavier** [NixOS/nixos-search#447]
See https://github.com/NixOS/nixpkgs/issues/164453: options like `networking.interfaces.<name>.ipAddress` are displayed (poorly) in nixos-search, while they are absent from the nixos manual.

We should either remove them, or make it clear that the option was renamed and link to the new option.

**Comment by garbas** [comment]
:+1: we should definitely show that an option was renamed and provide a link to new option.

@ncfavier Is this information exposed in our export json?

**Comment by ncfavier** [comment]
No, this would require changes to the nixpkgs module system to tag renamed options with the old option's name.

I probably won't have time to work on that before a while.

**Comment by garbas** [comment]
Np. Lets keep it in our backlog
